maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -185,7 +188,13 @@ private void loadProjectProperties(Builder builder, MavenProject project) throws
 
 		// Merge in current project properties
 		File baseDir = project.getBasedir();
-		File bndFile = new File(baseDir, Project.BNDFILE);
+		File bndFile;
+		if(bndLocation != null) {
+			bndFile = new File(bndLocation);
```

Comment:
Can't they just use an -include directive to achieve their goal of putting
the file where they like?